FirebaseAppDistribution/Sources/FIRAppDistribution.m

```diff
@@ -266,7 +266,7 @@ - (void)handleOauthDiscoveryCompletion:(OIDServiceConfiguration *_Nullable)confi
        appDistributionSignInCompletion:(void (^)(NSError *_Nullable error))completion {
   if (!configuration) {
     // TODO: Handle when we cannot get configuration
-    NSLog(@"ERROR - Cannot discover oauth config");
+    FIRFADInfoLog(@"ERROR - Cannot discover oauth config");
```

Comment:
Same question here

Reply:
We shouldn't need the ERROR prefix then.

Reply:
Yes, will switch these to error! :)